test/integration/framework/serializer.go

```diff
@@ -21,14 +21,6 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/serializer/versioning"
 )
 
-// NewSingleContentTypeSerializer wraps a serializer in a NegotiatedSerializer that handles one content type
-func NewSingleContentTypeSerializer(scheme *runtime.Scheme, info runtime.SerializerInfo) runtime.StorageSerializer {
-	return &wrappedSerializer{
-		scheme: scheme,
-		info:   info,
-	}
-}
-
 type wrappedSerializer struct {
```

Comment:
I think @deads2k question was if this type (and potentially the rest of the file) could also be removed since nothing is calling the constructor.